pkg/storage/unified/search/bleve.go

```diff
@@ -413,7 +414,11 @@ func toBleveSearchRequest(req *resource.ResourceSearchRequest, access authz.Acce
 		}
 	}
 
-	queries = append(queries, newTextQuery(req))
+	// Add a text query
+	if req.Query != "" && req.Query != "*" {
+		searchrequest.Fields = append(searchrequest.Fields, resource.SEARCH_FIELD_SCORE)
+		queries = append(queries, bleve.NewFuzzyQuery(req.Query))
```

Comment:
This only adds a query when one exists -- the rest already adds a matchAll wrapper